get-donor-data/get-donor-json-files.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+description: get the most recent donor json files
+version: 0.0.1
+created: 2018-02-21
+author: Ed Nykaza
+dependencies:
+    * requires environmental variables: import environmentalVariables.py
+    * list of donors generated by get-donor-list.py
+license: BSD-2-Clause
+TODO:
+* [] save files in the cloud
+"""
+
+# %% load in required libraries
+import environmentalVariables
+import pandas as pd
+import numpy as np
+import os
+import sys
+import subprocess as sub
+
+# %% user inputs (choices to be made to run code)
+securePath = "/tidepoolSecure/data/"
+dateStamp = "2018-02-27"
```

Comment:
Should this be hard coded? Do we always want the latest? Either way, should be a command line param if we need it.